mne/epochs.py

```diff
@@ -2132,9 +2132,18 @@ def _read_one_epoch_file(f, tree, fname, preload):
         if len(processed) == 0:
             raise ValueError('Could not find processed data')
 
-        epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
+        if fix_tags:
+            epochs_node = dir_tree_find(tree, 122)
+        else:
+            epochs_node = dir_tree_find(tree, FIFF.FIFFB_EPOCHS)
         if len(epochs_node) == 0:
-            raise ValueError('Could not find epochs data')
+            if fix_tags:
+                msg = 'Could not find epochs data'
+            else:
+                msg = ('Could not find epochs data. If the data is saved with '
+                       'old tagging system, the data can be read by using '
+                       'fix_tags option.')
+            raise ValueError(msg)
```

Comment:
Yeah exactly. We got lucky that the numbers we had previously assigned to e.g. FIFFB_EPOCHS and FIFFB_ICA (or whatever) don't correspond to entries we'd expect to see in the same places.